build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'io.jmix' version '1.0.0'
+    id 'java'
+}
+
+apply plugin: 'org.springframework.boot'
+
+jmix {
+    bomVersion = '1.0.0-SNAPSHOT'
+    projectId = 'mvcfmk'
+}
+
+group = 'com.company'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '1.8'
+
+repositories {
+    mavenCentral()
+    maven {
+        url 'https://nexus.jmix.io/repository/public'
+    }
+    maven {
+        url 'https://global.repo.jmix.io/repository/public'
+    }
+}
+
+ext {
+    mapstructVersion = '1.4.2.Final'
+}
+
+dependencies {
+    implementation 'io.jmix.core:jmix-core-starter'
+    implementation 'io.jmix.data:jmix-eclipselink-starter'
+    implementation 'io.jmix.ui:jmix-ui-starter'
+    implementation 'io.jmix.ui:jmix-ui-data-starter'
+    implementation 'io.jmix.ui:jmix-ui-themes-compiled'
+    implementation 'io.jmix.ui:jmix-ui-widgets-compiled'
+    implementation 'io.jmix.security:jmix-security-starter'
+    implementation 'io.jmix.security:jmix-security-ui-starter'
+    implementation 'io.jmix.security:jmix-security-data-starter'
+    implementation 'io.jmix.localfs:jmix-localfs-starter'
+
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-freemarker'
+
+    implementation "org.mapstruct:mapstruct:$mapstructVersion"
+    annotationProcessor "org.mapstruct:mapstruct-processor:$mapstructVersion"
```

Comment:
Let's make the sample as simple as possible. I mean that adding any code-generators is not required here. You can build DTOs using the constructor - just add a constructor with the User argument to the UserDto class. This will be the way to build UserDto. No separate Mapper classes are required in this case.